uktrade/great-cms
# KLS-533 - Fix styling issues for tablets on event list page

This PR makes three fixes to the styling on the events listing page:
1. Sets a min-width on the filter column, so that the filter text isn't cut off for tablets
2. Makes the event tag font size smaller for tablets so that the text doesn't go over the width of the box.
3. Adds a 5px padding one either side of the time text (e.g. `at 2:00pm`) so that the text doesn't touch the edge of the grey box (currently does between 670-700px)

**To test**
- Pull changes and go to http://greatcms.trade.great:8020/export-academy/events/
- Make sure that you have some events published in your local CMS to fully see the changes (they may be 'draft' by default)

**Screenshot before (641px width)**
![Screenshot 2023-04-24 at 10 53 30](https://user-images.githubusercontent.com/22460823/233963171-ff4c0a47-315f-42b9-b728-cb257f47db53.png)

**Screenshot after (641px width)**
![Screenshot 2023-04-24 at 10 53 49](https://user-images.githubusercontent.com/22460823/233963235-84be8a84-00cc-4adb-86aa-698272d6eab4.png)


### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-533

### Reviewing help


### Housekeeping


### Merging
